QuizApp/App/Common/Coordinator.swift

```diff
@@ -19,10 +19,16 @@ final class Coordinator: CoordinatorProtocol {
     }
 
     func showQuizDetails(quiz: QuizViewModel) {
-        let viewModel = QuizDetailsContainer.quizDetailsViewModel(quiz)
-        let viewController = QuizDetailsContainer.quizDetailsViewController(viewModel)
+        let viewModel = QuizDetailsViewModel(quiz: quiz, coordinator: self)
+        let viewController = QuizDetailsViewController(viewModel: viewModel)
         viewController.hidesBottomBarWhenPushed = false
         navigationController.pushViewController(viewController, animated: true)
     }
 
+    func showQuizSession(quiz: QuizViewModel) {
+        let viewModel = QuizSessionViewModel(quiz: quiz)
+        let viewController = QuizSessionViewController(viewModel: viewModel)
+        navigationController.pushViewController(viewController, animated: true)
+    }
+
```

Comment:
Use Factory to resolve

Reply:
Dont have it yet available. Need to merge it first. Will do it in the next PR, for the quiz session API.